docs/content/direct/hub-chart.md

```diff
@@ -0,0 +1,157 @@
+# KubeStellar Hub chart usage
+
+This documents explains how to use KubeStellar Hub chart to deploy a new instance of KubeStellar
+with a choise of user-defined Control Plains.
+
+The information provided is specific for the following release:
+
+```shell
+export KUBESTELLAR_VERSION=0.23.0-alpha.1
+```
+
+## Pre-requisites
+
+A list of executable that may be required during the deployment can be fund [here](./pre-reqs.md).
+At minimum [Helm](https://helm.sh/) is required.
+
+The setup of KubStellar via the Hub chart requires the existance of at least one cluster
+to be used for the deployment of the chart.
+
+This can be:
+
+1. A local **Kind** cluster with an ingress with SSL passthrough and a mapping to host port 9443
+
+This option is particularly useful for first time users or users that would like to have a local deployment.
+
+For convenience, a new local Kind clusters that satisfies the requirements for KubeStellar setup
+and that can be used to exercises the [examples](./examples.md) can be created with the following command:
+
+```shell
+sudo su <<EOF
+bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v${KUBESTELLAR_VERSION}/scripts/create-kind-cluster-with-SSL-passthrough.sh) --name kubestellar --port 9443

```

Comment:
If the port has to be 9443, and the script defaults to that value, then why repeat it here?

There was nothing said earlier about the `kind` cluster name. I suspect it has to be `kubestellar`, since I see no variable holding the cluster name. If this is right, then same question as for the port number.

Reply:
1. The intention was to "document" the options by exposing them to a user that may want to change them.
2. I rewarded the text below to explain that the port may be changed as well... and what needs to be done in that case
3. technically, there is bug in kflex that allows only `kubeflex`... until the next release of the chart... so I renamed back to `kubeflex`